project_task_stage_change_restriction/models/project_task.py

```
@@ -0,0 +1,58 @@
+        """
+        Raise UserError if the current user is not allowed to move
+        the task(s) to the given stage defined in vals['stage_id'].
+        """
+        if 'stage_id' not in vals:
+            return True
+
+        new_stage = self.env['project.task.type'].browse(vals['stage_id'])
+        for task in self:
+            if not self._is_move_allowed(task, new_stage, self.env.user):
+                raise UserError(_(
+                    "Sorry, you are not allowed to move the task '%s' into the stage '%s'."
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)